benchmark/infrastructure/BenchmarkResultToString.cpp

```diff
@@ -4,55 +4,196 @@
 
 #include "../benchmark/infrastructure/BenchmarkResultToString.h"
 
+#include <absl/strings/str_cat.h>
+#include <bits/ranges_algo.h>
+
+#include "BenchmarkMeasurementContainer.h"
+#include "BenchmarkMetadata.h"
+#include "util/Exception.h"
+#include "util/Forward.h"
+
 namespace ad_benchmark {
+
+extern const std::string outputIndention = "\t";
```

Comment:
This should be a `static constexpr std::string_view` directly in the header.
An please don't use tabs, use a fixed number of spaces (4 might be reasonable).

Reply:
Done, see [2e2d577](https://github.com/ad-freiburg/qlever/pull/958/commits/2e2d5775b14684dee66f4040ea5f13e26a905958).